src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/McPkg.cs

```diff
@@ -61,6 +69,7 @@ public McPkg(string plant, Project project, string commPkgNo, string mcPkgNo, st
         public ICollection<Certificate> CertificateScopes => _certificateScope;
         public DateTime CreatedAtUtc { get; private set; }
         public int CreatedById { get; private set; }
+        public Guid Guid { get; set; } = Guid.Empty;
```

Comment:
remove init Guid.Empty;
Add a Todo // make Guid private after FillGuids is completed